Source/WebCore/rendering/EventRegion.cpp

```diff
@@ -583,8 +589,44 @@ void EventRegion::uniteEventListeners(const Region& region, OptionSet<EventListe
         m_nonPassiveWheelEventListenerRegion.unite(region);
         LOG_WITH_STREAM(EventRegions, stream << " uniting for active wheel event listener");
     }
+#endif // ENABLE(WHEEL_EVENT_REGIONS)
+#if ENABLE(TOUCH_EVENT_REGIONS)
+    if (eventListenerRegionTypes.contains(EventListenerRegionType::TouchStart)) {
+        m_touchEventListenerRegion.start.unite(region);
```

Comment:
I'm not sure that we have to track an event for each type of touch event. Talk to Wenson to see what we need in the UI process.